aries-site/src/examples/components/layer/LayerCenterExample.js

```diff
@@ -0,0 +1,51 @@
+import React, { useContext, useState } from 'react';
+import { Button, Box, Heading, Layer, ResponsiveContext, Text } from 'grommet';
+import { Alert, MailOption } from 'grommet-icons';
+
+export const LayerCenterExample = () => {
+  const [open, setOpen] = useState(false);
+  const size = useContext(ResponsiveContext);
+  const onOpen = () => setOpen(true);
+  const onClose = () => setOpen(undefined);
+  return (
+    <>
+      <Box align="start">
+        <Button label="Show me the Layer" onClick={onOpen} primary />
+      </Box>
+      {open && (
+        <Layer position="center" modal onClickOutside={onClose} onEsc={onClose}>
```

Comment:
Just wondering if there was a reason we didnt add an icon to exit

Reply:
Good question. I didn't put one here because there wasn't one on the design. Greg mentioned that there may be cases where a "X" may not be needed and that some expanded guidance would be needed.

I think we needed more guidance on when it is okay to not have an "X" button

Reply:
A follow up here ^ The design here is still in discussion and guidance will be provided on design is solidified. But for now, we will go with this.